fineract-provider/src/main/java/org/apache/fineract/infrastructure/documentmanagement/contentrepository/S3ContentRepository.java

```diff
@@ -154,23 +154,19 @@ private void deleteObject(final String location) {
     private void putObject(final String filename, final InputStream inputStream, final String s3UploadLocation)
             throws ContentManagementException {
         try {
-            LOG.info("Uploading a new object to S3 {}", s3UploadLocation);
+            LOG.info("Uploading a new object to S3 {}", s3UploadLocation.replaceAll("[\n\r\t]", "_"));
             this.s3Client.putObject(new PutObjectRequest(this.s3BucketName, s3UploadLocation, inputStream, new ObjectMetadata()));
-        } catch (AmazonServiceException ase) {
+        } catch (AmazonClientException ase) {
```

Comment:
AmazonServiceException and AmazonClientException are two distinct exceptions with no hierarchical relation. Are you sure we're not gonna need AmazonServiceException here?
Just to be on the safe side not to break anything, I'd say let's go with catching both in a multi-catch block like
```
} catch (AmazonClientException |  AmazonServiceException ase) {
```